reports/report_builder.py

```diff
@@ -648,12 +648,28 @@ def ws_05(self, ws):
                     counts.update({(r['sex'], TOPIC_GROUPS[r['topic']]): r['n']})
 
             counts_list.append(secondary_counts)
+        overall_column = ws.dim_colmax
         self.tabulate_secondary_cols(ws, counts_list[0], self.male_female, MAJOR_TOPICS, row_perc=True)
         c = ws.dim_colmax + 2
+        overall_row = ws.dim_rowmax+2
+        for media_type in counts_list[0]:
+            overall_total_dm = sum([counts_list[0][media_type][x] for x in counts_list[0][media_type]])
+            female_values = [x for x in counts_list[0][media_type] if x[0]==self.female_ids[0]]
+            overall_tm_female_value = sum([counts_list[0][media_type][x] for x in female_values]) / overall_total_dm
```

Comment:
```suggestion
            overall_tm_female_value = sum([counts_list[0][media_type][x] for x in female_values]) / overall_total_tm
```

Mixing `tm` and `dm` I think